nemo_skills/pipeline/utils.py

```diff
@@ -782,6 +944,10 @@ def get_cluster_config(cluster=None, config_dir=None):
 
     cluster_config = read_config(config_file)
 
+    # resolve ssh tunnel config
```

Comment:
should we move this to get_cluster_config function?

Reply:
This is part of get_cluster_config() ?

Reply:
oh, sorry, you're right